Sources/PackageModel/SwiftSDKs/SwiftSDKConfigurationStore.swift

```diff
@@ -202,6 +202,7 @@ public final class SwiftSDKConfigurationStore {
             }
 
             if showConfiguration {
+                print("\nSDK: \(targetTriple)")
```

Comment:
IMO this should be hidden under a flag. There are scripts in existence relying on each output line being a valid Swift SDK ID, and this change needlessly breaks this workflow without any warning or means for transition.

Reply:
Hmm, are you sure there are many script consumers yet? I suspect that most users of these commands are on the command-line to simply show these SDKs, and this is very important info to everyone, that follows the general format of the existing output. I suspect the few script consumers will be very happy to have this important info and will quickly update their scripts.